app/uk/gov/hmrc/hec/models/CorrectiveAction.scala

```diff
@@ -25,8 +25,8 @@ import play.api.libs.json.Format
 sealed trait CorrectiveAction extends EnumEntry with Product with Serializable
 
 object CorrectiveAction extends Enum[CorrectiveAction] {
-  case object Register extends CorrectiveAction
-  case object Dormant extends CorrectiveAction
+  case object RegisterNewSaAccount extends CorrectiveAction
```

Comment:
Should we make it SA in caps?

Reply:
yeah, I can do that :+1:

Reply:
updated [here](https://github.com/hmrc/hec/pull/50/commits/7fe0f099c70309f6037af55a95b8181706e84d3e)